h2o-docs/src/product/automl.rst

```diff
@@ -468,51 +468,103 @@ Information about how to cite the H2O software in general is covered in the `H2O
 Random Grid Search Parameters
 -----------------------------
 
-AutoML performs hyperparameter search over a variety of H2O algorithms in order to deliver the best model. In AutoML, the following hyperparameters are supported by grid search.  Random Forest and Extremely Randomized Trees are not grid searched (in the current version of AutoML), so they are not included in the list below.
+AutoML performs a hyperparameter search over a variety of H2O algorithms in order to deliver the best model. In the table below, we list the hyperparameters, along with all potential values that can be randomly chosen in the search. If these models also have a non-default value set for a hyperparameter, we identify it in the list as well. Random Forest and Extremely Randomized Trees are not grid searched (in the current version of AutoML), so they are not included in the list below.
 
 
-**GLM Hyperparameters**
-
--  ``alpha``
--  ``missing_values_handling``
-
-
-**XGBoost Hyperparameters**
-
--  ``ntrees``
--  ``max_depth``
--  ``min_rows``
--  ``min_sum_hessian_in_leaf``
--  ``sample_rate``
--  ``col_sample_rate``
--  ``col_sample_rate_per_tree``
--  ``booster``
--  ``reg_lambda``
--  ``reg_alpha``
-
-**GBM Hyperparameters**
-
--  ``histogram_type``
--  ``ntrees``
--  ``max_depth``
--  ``min_rows``
--  ``learn_rate``
--  ``sample_rate``
--  ``col_sample_rate``
--  ``col_sample_rate_per_tree``
--  ``min_split_improvement``
-
+GLM Hyperparameters
+~~~~~~~~~~~~~~~~~~~
 
-**Deep Learning Hyperparameters**
+This table shows the GLM values that are searched over when performing AutoML grid search. Additional information is available `here <https://github.com/h2oai/h2o-3/blob/master/h2o-automl/src/main/java/ai/h2o/automl/modeling/GLMStepsProvider.java>`__.
+
++-----------------------------+---------------------------------------------------------------------------------------------+
+| Parameter                   | Searchable Values                                                                           |
++=============================+=============================================================================================+
+| ``alpha``                   | ``{0.0, 0.2, 0.4, 0.6, 0.8, 1.0}``                                                          |
```

Comment:
technically, we don't run a grid search anymore for GLM, but we build a single model with `lambda_search` enabled, and passing a list of `alpha`s.
GLM algo is supposed to do lambda search for each `alpha` and return only the model with the best `alpha-lambda` combination in that case, whereas a grid would return one model for each `alpha`.